SystemInformer/proctree.c

```diff
@@ -2932,27 +2921,8 @@ BOOLEAN NTAPI PhpProcessTreeNewCallback(
 #endif
                 break;
             case PHPRTLC_ELEVATION:
-                {
-                    PWSTR type;
-
-                    switch (processItem->ElevationType)
-                    {
-                    case TokenElevationTypeDefault:
-                        type = L"N/A";
-                        break;
-                    case TokenElevationTypeLimited:
-                        type = L"Limited";
-                        break;
-                    case TokenElevationTypeFull:
-                        type = L"Full";
-                        break;
-                    default:
-                        type = L"N/A";
-                        break;
-                    }
-
-                    PhInitializeStringRefLongHint(&getCellText->Text, type);
-                }
+                getCellText->Text = PhGetStringRef(PhGetElevationTypeString(processItem->IsElevated ? TRUE : FALSE, processItem->ElevationType));
+                PhDereferenceObject(&getCellText->Text);
```

Comment:
@jxy-s: is this right now or i'm still totally wrong with leaking the `PH_STRING`; i belive i have seen in a textfile in the past, how is dealing with strings, but i currently i dont find it anymore.